pyproject.toml

```diff
@@ -25,6 +25,8 @@ dependencies = [
     'typing-extensions>=4.10.0; python_version<"3.13"',
     "scikit-video>=1.1.11",
     "scipy>=1.13.0",
+    "ntplib>=0.4.0",
+    "pillow>=11.0.0,<11.3.0",
```

Comment:
Why is pillow added here? I don't see it used

Reply:
Good catch, don't need it in this PR. Got mixed up while working on this at the same time, trying a bunch of camera-detection stuff that works on Windows too (which I gave up for now).
https://github.com/miniscope/mio/pull/144

 I think I ended up not needing `pillow` for this, too, so I will remove it.